services/nublado2/values-bleed.yaml

```diff
@@ -0,0 +1,49 @@
+nublado2:
+  jupyterhub:
+    debug:
+      enabled: true
+
+    hub:
+      image:
+        name: lsstsqre/nublado2
+        tag: "1.0.0"
+
+    singleuser:
+      storage:
+        type: none
+
+    auth:
+      state:
+        cryptoKey: "080c6d0eb8d9f84ef5f0860bea71dba6ab2e1be712c7bc8c2eaf92321c9fd968"
+
+    proxy:
+      secretToken: "3854245f5b48412ee9b7a2c985da6acdc146f80cc4f10eb175f0efa24530399b"
```

Comment:
These are secrets, which is okay as a temporary stopgap for minikube, but bleed is on the Internet. I think they need to be real secrets before we add them to non-minikube environments.

Reply:
Ah yes that's right, I was working on adding cachemachine secrets generation to the generate secrets.py, but I couldn't finish that before I left, and was tempted to just do it without that and add it later, but I'll try to add that in.  My problem got around ingested a dockerconfigjson file as a secret... it seems to work fine with the vault command line, but putting it in a json document doesn't seem to parse... I might have to work on that one for a bit.